library/src/scala/annotation/publicInBinary.scala

```diff
@@ -0,0 +1,14 @@
+package scala.annotation
+
+/** A binary API is a definition that is annotated with `@publicInBinary` or overrides a definition annotated with `@publicInBinary`.
+ *  This annotation can be placed on `def`, `val`, `lazy val`, `var`, `object`, and `given` definitions.
+ *  A binary API will be publicly available in the bytecode.
+ *
+ *  This annotation cannot be used on `private`/`private[this]` definitions.
+ *
+ *  This can be used to access private[T]/protected definitions within inline definitions.
+ *
+ *  Removing this annotation from a non-public definition is a binary incompatible change.
```

Comment:
Adding it is also binary incompatible since it means some accessors won't be generated, right?

Reply:
Yes. Added doc.